pom.xml

```diff
@@ -93,7 +93,7 @@
 
         <!-- Test Dependency versions -->
         <mockito-all.version>1.8.5</mockito-all.version>
-        <junit.version>4.12</junit.version>
+        <junit.version>4.13.1</junit.version>
```

Comment:
I think there is 4.13.2 version for junit, is there a reason why we don't use that?

Reply:
I wasn't aware of it.
Thanks.

Reply:
Update to 4.13.2